### PR TITLE
Benchmark running for druid-0.10.0

### PR DESCRIPTION
**Pull request by rcmgleite**
- [x] Fixed benchmark-druid.R for min/max aggregations
- [x] Fixed ingestion task descriptors

### CODE REVIEW

**Comment by Aaaaaaron** [comment]
@rcmgleite the configuration is also outdated

File: lineitem.task.json [hunk 1]
```diff
@@ -1,91 +1,100 @@
...
-      "targetPartitionSize": 5000000
-    },
-    "pathSpec": {
-      "type": "static",
-      "paths": "s3:\/\/path\/to\/tpch\/data\/lineitem.tbl.*.gz"
-    },
-    "rollupSpec": {
-      "aggs": [
+    "dataSchema": {
+      "dataSource": "tpch_lineitem_small",
```
Comment (Aaaaaaron) [inline lineitem.task.json:12]: should be tpch_lineitem

File: lineitem.task.json [hunk 2]
```diff
@@ -1,91 +1,100 @@
...
+      "granularitySpec": {
+        "type": "uniform",
+        "intervals": ["1992-01-01T00:00:00/1999-01-01T00:00:00"],
+        "queryGranularity": "none",
+        "segmentGranularity" : "day"
```
Comment (Aaaaaaron) [inline lineitem.task.json:17]: month?

File: lineitem.task.json [hunk 3]
```diff
@@ -1,91 +1,100 @@
...
-      "paths": "s3:\/\/path\/to\/tpch\/data\/lineitem.tbl.*.gz"
-    },
-    "rollupSpec": {
-      "aggs": [
+    "dataSchema": {
+      "dataSource": "tpch_lineitem_small",
+      "granularitySpec": {
+        "type": "uniform",
+        "intervals": ["1992-01-01T00:00:00/1999-01-01T00:00:00"],
+        "queryGranularity": "none",
```
Comment (Aaaaaaron) [inline lineitem.task.json:16]: should be day?

**Comment by rcmgleite** [comment]
@tttMelody I don't think this project is supported anymore..
I've opened this PR more than a year ago..

**Pull request closed without merging**